packages/discovery-react-components/src/components/SearchResults/SearchResults.tsx

```diff
@@ -256,6 +273,45 @@ const SearchResults: React.FunctionComponent<SearchResultsProps> = ({
   );
 };
 
+/**
+ * Hook to update search parameters' `return` param with the minimal necessary fields
+ * that are necessary to render this component.
+ *
+ * NOTE: This hook must be specified before any other use of `setSearchParameters`.
```

Comment:
Should we create a hook that does both and start using that from now on?

Reply:
Re-reading the comment, and I'm wondering if my initial read was what you meant. Does this have to run once and only once before using `setSearchParameters` or before each and every use of `setSearchParameters`?

Reply:
This hook will update search params any time they change. I went ahead and removed that note, though, since I don't think that's true any more.